backend/modules/parsers/multimodalparser.py

```diff
@@ -59,29 +59,23 @@ class MultiModalParser(BaseParser):
 
     supported_file_extensions = [".pdf", ".png", ".jpeg", ".jpg"]
 
-    def __init__(
-        self,
-        *args,
-        **kwargs,
-    ):
+    def __init__(self, *, model_configuration: dict = {}, prompt: str = "", **kwargs):
```

Comment:
```suggestion
    def __init__(self, *, model_configuration: ModelConfig = None, prompt: str = "", **kwargs):
```

1. We should not use mutable defaults
2. Let the type indicate we want `ModelConfig` instance


Line 69 will ensure we have converted to ModelConfig even if get a dict

Reply:
This is one of the reasons why I want to move base classes to pydantic

so pydantic can take care of casting and conversion and we get schema for free

Reply:
Sure.